src/index.html

```diff
@@ -11,8 +11,66 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
The href attribute should link to a CSS file, not an SCSS file. Ensure that the compiled CSS file is linked here instead of the SCSS source file.